backend/server.js

```diff
@@ -9,16 +9,33 @@ app.use(cors());
 app.use(bodyParser.json());
 
 const JWT_Secret = 'secret_key';
-let testUser = { email: 'Valenteen@gmai.com', password: '1234'};
 
-app.post('/api/authenticate', (reg,res) => {
+const users = [
+    { email: 'Valenteen@gmai.com', password: '1234'},
+    { email: 'Admin@gmai.com', password: '4321'}
+];
+
+app.post('/api/authenticate', (reg, res) => {
     if(reg.body) {
-        let user = reg.body;
+        const user = reg.body;
+        let isPermission = false;
+        let isEmail = false;        
+        
+        users.forEach((item) => {
```

Comment:
It is very bad. I can't understand how it works

You can find user using 'find' method of array and then check what you need already found user